xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/PropAddAction.java

```diff
@@ -80,13 +81,16 @@ public boolean action(XWikiContext context) throws XWikiException
         BaseClass bclass = doc.getXClass();
         bclass.setName(doc.getFullName());
         if (bclass.get(propName) != null) {
```

Comment:
The content of this if seems to be mostly duplicating `writeAjaxErrorResponse`.

Reply:
Thanks, I couldn't find it. This is definitely doing the same thing.